lib/image-stream/handlers.js

```diff
@@ -123,6 +124,46 @@ module.exports = {
   },
 
   /**
+   * @summary Handle Apple disk images (.dmg)
+   * @function
+   * @public
+   * @memberof handlers
+   *
+   * @param {String} file - file path
+   * @param {Object} options - options
+   * @param {Number} [options.size] - file size
+   *
+   * @fulfil {Object} - image metadata
+   * @returns {Promise}
+   */
+  'application/x-apple-diskimage': (file, options) => {
+    return new Bluebird((resolve, reject) => {
+      const dmg = new UDIF.Image(file);
```

Comment:
Hahaha 😁  I'm thinking I could just stick the open -> getUncompressedSize -> close into a method on UDIF, i.e. `UDIF.getUncompressedSize( filename, callback )` and then we can avoid doing this dance altogether :)